lib/pageFetcher.ts

```diff
@@ -93,12 +93,18 @@ export class Fetcher {
         this.closed = true;
     }
 
-    async fetch<S>(node: Node, state: S, notifier: Notifier<FetchEvent, S>) {
+    async fetch<S>(node: Node, state: S, notifier: Notifier<FetchEvent, S>, includeMetadata: boolean) {
         try {
-            const resp = await this.dereferencer.dereference(node.target, {
+            const options: IDereferenceOptions = {
                 localFiles: true,
                 fetch: this.fetch_f,
-            });
+            };
+            if (includeMetadata) {
+                options.headers = {
+                    Accept: "application/metadata+trig",
```

Comment:
Hm this overrides the normal accept header right?
This will probably result in issues when the server doesn't accept application/metadata+trig.
Could you either, find a solution to append application/metadata+trig or add reasonable defaults with text/turtle etc as well?